compose/.env.es6

```diff
@@ -14,6 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ES_VERSION=6.8.8
+ES_VERSION=6.8.18
```

Comment:
Confirmed, https://github.com/elastic/elasticsearch/blob/v6.8.18/LICENSE.txt 6.8.18 is still Apache 2.0 License.